client/dashboard/profile-wizard/steps/benefits/index.js

```diff
@@ -202,87 +187,60 @@ class Benefits extends Component {
 		}
 		const pluginNamesString = pluginsToInstall
 			.map( ( pluginSlug ) => pluginNames[ pluginSlug ] )
-			.join( ' & ' );
+			.join( ' ' + __( 'and', 'woocommerce-admin' ) + ' ' );
```

Comment:
@rrennick Do you think this will work okay for translation?

Reply:
>Do you think this will work okay for translation?

That works okay. Only the `'and'` is passed through translation. The spaces are concatenated after translation.